recipes/gFlex/meta.yaml

```
@@ -0,0 +1,46 @@
+{% set name = "gflex" %}
+{% set version = "1.1.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://github.com/awickert/gFlex/archive/v1.1.0.tar.gz
+  sha256: dc722606dbc6a549c84bc86849419bc43e4fcd9c0d8329984fe6fbf6b0b209d4
+
+
+build:
+  number: 0
+  entry_points:
+    - gflex = gflex:main
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - python
+    - scipy
+    - matplotlib
+
+test:
+  imports:
+    - gflex
+  commands:
+    - gflex --help
```

Can we try to add ``pip check`` here please?
```suggestion
    - gflex --help
    - pip check
  requires:
    - pip
```